hazelcast-spring-tests/src/test/java/com/hazelcast/spring/TestFullApplicationContext.java

```diff
@@ -371,15 +373,20 @@ public void testMapConfig() {
         assertEquals(321, journalConfig.getTimeToLiveSeconds());
         assertEquals(MetadataPolicy.OFF, testMapConfig.getMetadataPolicy());
         assertTrue(testMapConfig.isReadBackupData());
-        assertEquals(2, testMapConfig.getIndexConfigs().size());
+        assertEquals(3, testMapConfig.getIndexConfigs().size());
         for (IndexConfig index : testMapConfig.getIndexConfigs()) {
             if ("name".equals(index.getAttributes().get(0))) {
                 assertEquals(IndexType.HASH, index.getType());
                 assertNull(index.getName());
-            } else if ("age".equals(index.getAttributes().get(0))) {
+            } else if ("sortedIndex".equals(index.getName())) {
                 assertEquals(IndexType.SORTED, index.getType());
-                assertEquals("sortedIndex", index.getName());
-                assertEquals("name", index.getAttributes().get(1));
```

Comment:
minor: can we add `assertEquals("name", index.getAttributes().get(1));` assertion back?